sentry_sdk/integrations/django/__init__.py

```diff
@@ -357,6 +359,38 @@ def sentry_patched_get_response(self, request):
         patch_get_response_async(BaseHandler, _before_get_response)
 
 
+def patch_templates():
+    # type: () -> None
+    import django.shortcuts
+    from django.template.response import SimpleTemplateResponse
+
+    real_render = django.shortcuts.render
+    real_rendered_content = SimpleTemplateResponse.rendered_content
+
+    @_functools.wraps(real_render)
+    def render(request, template_name, context=None, *args, **kwargs):
+        hub = Hub.current
+        if hub.get_integration(DjangoIntegration) is None:
+            return real_render(request, template_name, context, *args, **kwargs)
+
+        with hub.start_span(op="django.render", description=template_name) as span:
+            span.set_data("context", context)
+            return real_render(request, template_name, context, *args, **kwargs)
+
+    @property
+    def rendered_content(self):
```

Comment:
Are invocations of `rendered_content` and `render` mutually exclusive? I have concerns that we will have two `django.render` spans in one transaction without a way to distinguish them.

Reply:
yes they are exclusive, one render in the view span, the template response is lazy and rendered out of the view automatic span.